src/signals/IncidentMap/components/Header/header.tsx

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { Fragment } from 'react'
+
+import {
+  Hidden,
+  MenuButton as Button,
+  MenuItem as Item,
+  Heading,
+  MenuToggle,
+} from '@amsterdam/asc-ui'
+
+import Logo from 'components/Logo'
+
+import { HeaderWrapper, Title } from './styled'
+
+export const Header = () => {
+  const MenuItems = () => (
+    <Item>
+      <Button forwardedAs="a" href="/incident/beschrijf" target="_blank">
+        Doe een melding
+      </Button>
+    </Item>
+  )
+
+  const Navigation = () => (
```

Comment:
only question, why is this declared here instead of put into the main jsx?

Reply:
Refactored the component slightly to use media query like we use in the SiteHeader as well. The Hidden component was difficult to test.